services/src/main/java/org/jboss/windup/web/services/rest/MigrationProjectEndpointImpl.java

```diff
@@ -76,6 +77,9 @@ public MigrationProject getMigrationProject(Long id)
     @Override
     public MigrationProject createMigrationProject(MigrationProject migrationProject)
     {
+        if (null != getProjectIdByName(migrationProject.getTitle()))
```

Comment:
I don't see how it was even close to "trolling". It isn't my taste, but I don't see that as a blocker as we already have this idiom elsewhere. I think @klinki's comment was just a lighthearted joke, since they are actually called "yoda" conditions.